test/Microsoft.ML.Benchmarks/Numeric/Ranking.cs

```diff
@@ -85,7 +85,7 @@ public void SetupScoringSpeedTests()
             if (!File.Exists(_mslrWeb10k_Train))
                 throw new FileNotFoundException(string.Format(Errors.DatasetNotFound, _mslrWeb10k_Train));
 
-            _modelPath_MSLR = Path.Combine(Directory.GetCurrentDirectory(), @"FastTreeRankingModel.zip");
+            _modelPath_MSLR = Path.Combine(Path.GetDirectoryName(typeof(RankingTest).Assembly.Location), "FastTreeRankingModel.zip");
```

Comment:
Why is this change necessary?

Reply:
otherwise on .netfx it creates this new model file in the root directory

Reply:
This confuses me. What do you mean by "the root directory"? Why would the current netfx have its current directory set to that?

also, the code is still using Directory.GetCurrentDirectory() in `Test_Multiclass_WikiDetox_BigramsAndTrichar_OVAAveragedPerceptron`. WHy doesn't that need to change?

Reply:
> Why would the current netfx have its current directory set to that?

I don't have nay concrete explaination for why it is setting current directory to ```C:\git\Mlnet\test\Microsoft.ML.Benchmarks```

Reply:
@adamsitnik - do you happen to know? Is this something BenchmarkDotNet is controlling?

Reply:
@eerhardt  Yes BDN sets the working directory to the folder with built dlls for .NET Core and to null for FX.  The problem is that I dont remember why and I would prefer not to change it if I dont have to ;) It most probably has to deal with the fact that we run executable for FX and do `dotnet name.dll` for Core

https://github.com/dotnet/BenchmarkDotNet/blob/573566b70f0ff27572b9268002fe9748be4de6ea/src/BenchmarkDotNet/Toolchains/DotNetCli/DotNetCliExecutor.cs#L72

https://github.com/dotnet/BenchmarkDotNet/blob/573566b70f0ff27572b9268002fe9748be4de6ea/src/BenchmarkDotNet/Toolchains/Executor.cs#L35